### PR TITLE
docs: Add EBS CSI add-on to Operator installation tutorial

### LINKED ISSUE #901: Namespaces not consistent in EKS install instructions
### Describe the Issue

https://docs.redpanda.com/docs/platform/reference/redpanda-operator/kubernetes-qs-cloud/#amazon-eks-prerequisites uses "redpanda-operator" as the namespace for installing the operator, but "panda-chat" as the namespace for the actual cluster.  I think both of these need to be the same namespace; at least these instructions didn't work for me.

### PR DESCRIPTION
**Pull request by JakeSCahill**
Review deadline: January 31

Also:

- Made tabs keep in sync, so if you select one the others change.
- Fixed indentations.
- Fixed #901 

@tmgstevens please could you test that installing the add-on fixes your issue?

### CODE REVIEW

**Comment by tmgstevens** [comment]
Thanks for updating this @JakeSCahill and apologies for the back and forth!

File: docs/reference/redpanda-operator/kubernetes-qs-cloud.mdx [hunk 1]
```diff
@@ -73,58 +73,99 @@ gcloud container clusters create redpanda --machine-type n1-standard-4 --num-nod
...
+        name: redpanda
+        region: us-west-1
+      nodeGroups:
+        - name: standard-workers
+          desiredCapacity: 1
+          minSize: 1
+          maxSize: 2
+          instanceType: m5.xlarge
+          preBootstrapCommands:
+            - "curl -1sLf 'https://dl.redpanda.com/nzc4ZYQK3WRGd9sy/redpanda/cfg/setup/bash.rpm.sh' | sudo -E bash && sudo yum -y install redpanda"
```
Comment (Feediver1) [inline docs/reference/redpanda-operator/kubernetes-qs-cloud.mdx:117]: Take a look at the wrapping in the rendered output here @JakeSCahill . Is this what you expected?

File: docs/reference/redpanda-operator/kubernetes-qs-cloud.mdx [hunk 2]
```diff
@@ -73,58 +73,99 @@ gcloud container clusters create redpanda --machine-type n1-standard-4 --num-nod
...
+
+      ```bash
+      eksctl create cluster -f eks-bootstrap.yaml
+      ```
+
+      :::note
+      If your account is configured for OIDC, add the `--with-oidc` flag to the `create cluster` command.
+      :::
+
+1. Create the IAM role needed for the Amazon Elastic Block Store (EBS) Cluster Storage Interface (CSI):
```
Comment (Feediver1) [inline docs/reference/redpanda-operator/kubernetes-qs-cloud.mdx:137]: So I am a bit confused about the numbering here...this is 1, but there is not a new heading to differentiate this from the previous step 1 above (create an Amazon EKS cluster). Shouldn't this be step 2?

File: docs/reference/redpanda-operator/kubernetes-qs-cloud.mdx [hunk 3]
```diff
@@ -73,58 +73,99 @@ gcloud container clusters create redpanda --machine-type n1-standard-4 --num-nod
...
+1. Add the EBS CSI add-on:
 
-```bash
-eksctl create cluster -f eks-bootstrap.yaml
-```
+  ```bash
+  eksctl create addon \
+      --name aws-ebs-csi-driver \
+      --cluster redpanda \
+      --service-account-role-arn arn:aws:iam::${AWS_ACCOUNT_ID}:role/AmazonEKS_EBS_CSI_DriverRole \
```
Comment (Feediver1) [inline docs/reference/redpanda-operator/kubernetes-qs-cloud.mdx:162]: Check the formatting here--line break char is rendering at the beginning of the block.

**Comment by JakeSCahill** [comment]
Thanks @Feediver1 !

**Pull request merged**